src/m365/spo/commands/web/web-get.ts

```diff
@@ -25,9 +26,17 @@ class SpoWebGetCommand extends SpoCommand {
     return 'Retrieve information about the specified site';
   }
 
+  public getTelemetryProperties(args: CommandArgs): any {
+    const telemetryProps: any = super.getTelemetryProperties(args);
+    telemetryProps.withGroups = typeof args.options.withGroups !== 'undefined';
+    return telemetryProps;
+  }
+
   public commandAction(logger: Logger, args: CommandArgs, cb: () => void): void {
     const requestOptions: any = {
-      url: `${args.options.webUrl}/_api/web/`,
+      url: !args.options.withGroups ?
```

Comment:
Rather than duplicating the first portion of the URL, let's define it on a `let` and conditionally add the expand part

Reply:
Now the code is neater.. Awesome. Thanks... 👍